test/integration/redis_proxy_integration_test.cc

```diff
@@ -0,0 +1,137 @@
+#include <sstream>
```

Comment:
Apologies, I should have caught this in the last round, but this test should go in the extension folder (we do this for integration tests specific to an extension): https://github.com/envoyproxy/envoy/tree/master/test/extensions/filters/network/redis_proxy

The test can then be guarded like the unit tests to only run when redis is compiled.

/wait

Reply:
I should have caught that myself...